### ISSUE TITLE
id=gleeModalContainer missing quotation marks

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#14]
id=gleeModalContainer missing quotation marks
==> add them. needs to be id="gleeModalContainer".
